scripts/main.js

```diff
@@ -52,11 +52,11 @@ module.exports = (robot) => {
         }
     })
     robot.error((err, res) => {
-        console.log(err)
-        robot.logger.error("DOES NOT COMPUTE")
+        console.log(err.message)
+        robot.logger.error(err.message)
```

Comment:
@Mikanebu you can do whatever you think is right, just "DOES NOT COMPUTE" does not give me any useful information, at least say "Some thing went wrong" or something, you just asked for a suggestion and I just gave one. 

But even so, I would test that it is working in case of error. Just to be sure that it is acting as expected in case of error